corehq/apps/userreports/models.py

```diff
@@ -205,6 +206,72 @@ def is_rebuilding_in_place(self):
     def is_rebuild_in_progress(self):
         return self.is_rebuilding or self.is_rebuilding_in_place
 
+    def rebuild_failed(self, data_source_config_id):
+        """
+        Returns ``True`` if the rebuild failed, ``False`` if it succeeded
+        or has not yet failed, or ``None`` if Flower is not available.
+        """
+        flower_url = getattr(settings, 'CELERY_FLOWER_URL')
```

Comment:
unlike `{}.get`, `getattr` requires a fallback or it raises `AttributeError`
```suggestion
        flower_url = getattr(settings, 'CELERY_FLOWER_URL', None)
```
easy to workaround in dev though - I just added it to my localsettings